src/module/vue/components/xp-box.vue

```diff
@@ -10,8 +10,6 @@ const props = defineProps<{ value: number; current: number }>()
 
 const classes = computed(() => {
   return {
-    clickable: true,
-    block: true,
```

Comment:
These lost their hover state:


https://user-images.githubusercontent.com/39902/196040521-7d57d838-26c6-490c-b6fb-7a6c89b6eefb.mp4

Reply:
yeah, my plan was to restore that before removing the draft status. the method to my madness here was roughly:

* [x] draw up a palette of colour variables, organized by attributes of the colour ("light", "dark", etc)
* [x] migrate all styles specific to a single component to the corresponding SFC
* [x] standardize component css variables
  * root out anything that looks like a good CSS variable candidate and replace it with one
  * synthesize formal standards/patterns from existing *de facto* standards/patterns where noticed, taking care to maintain their coherence when considered as a group
  * develop new variables as needed, and organize by purpose or semantics of the variable (referencing the palette vars, above)
  * periodically stop to re-evaluate their semantics/organization
* [ ] (partially complete) pick over the rendered sheet, adjusting values until it renders at parity (though not necessarily identically) with the original
* [ ] review and prune CSS variables, removing ones that are underutilized, etc
* [ ] review PR's diffs, culling changes that are trivial or deferrable, plus any placeholder code, fixable FIXMEs, or weird artifacts
* [ ] untick PR's draft status